tst/EnergyPlus/unit/SetPointManager.unit.cc

```diff
@@ -306,7 +306,7 @@ TEST_F(EnergyPlusFixture, SetPointManager_DefineCondEntSetPointManager)
                                                       "Dimensionless;           !- Input Unit Type for z",
                                                       "Curve:QuadLinear,",
                                                       "OptCondEntCurveName,     !- Name",
-                                                      "12.2711,                 !- Coefficient1 Constant",
+                                                      "13.2711,                 !- Coefficient1 Constant",
```

Comment:
Unit test broke after the curve bugfix. Adjusted the constant to make it passing.

Reply:
:+1: